### DISCUSSION TITLE
How can I get the request time?

### DISCUSSION THREAD

**Discussion by Wedrix** [walkor/Workerman#1024] [General]
How can I get the request time like with PHP's $_SERVER['REQUEST_TIME']?
I'm currently considering setting it myself in the Request class (i'm using my own Request class) but was wondering there's already some inbuilt API for this? For instance, you can get the ip address from the Connection [$request->connection->getRemoteIp()].
I'm using v4.

**Comment by walkor** [comment]
Workerman does not have an interface to get the request time. You can implement it by yourself.
